yb-voyager/cmd/exportDataStatus.go

```diff
@@ -54,6 +57,15 @@ func initializeExportTableMetadata(tableList []*sqlname.SourceName) {
 		tablesProgressMetadata[key].Status = 0
 		tablesProgressMetadata[key].FileOffsetToContinue = int64(0)
 		tablesProgressMetadata[key].TableName = tableName
+		exportSnapshotStatus.Tables[key] = &srcdb.TableExportStatus{
+			TableName:                tableName.Qualified.MinQuoted,
+			FileName:                 "",
+			ExportedRowCountSnapshot: int64(0),
+		}
+	}
+	err := exportSnapshotStatus.Create()
```

Comment:
Will this work if `export data` is called multiple times?

Reply:
yeah, deleting it on start-clean so create will work on every run of export data which can happen with start-clean only.